Sources/BraintreeCore/Date+MillisecondTimestamp.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+extension Date {
+    
+    var utcTimpestampMilliseconds: Int {
```

Comment:
We use this in a few places so I moved it into an extension. If you hate it, feel free to revert lol